### DISCUSSION TITLE
Youtube Links

### DISCUSSION THREAD

**Discussion by rmwood721** [requarks/wiki#3430] [Help / Questions]
**Question**
Why is my youtube link not appearing on my wiki.js page? It's appearing in the editor, but not on the saved page.

**Host Info (please complete the following information):**
OS: Mac OSX
Wiki.js version: 2.4.107
Database engine: Not sure

**Comment by Jhobean** [comment]
This issues is the same if the one? https://github.com/Requarks/wiki/issues/1459 . If yes you can close it

**Comment by rmwood721** [comment]
I’ll try that. There are comments further down that say it doesn’t work, but I’ll give it a shot.

Russell M. Wood, CSM
Informatics - Technical Writer
Zoobook Systems, LLC
732.707.7780
800.995.6997
russell@zoobooksystems.com

**Comment by CleverMindDev** [comment]
If interested, there is a PR which fix the issue.
See: https://github.com/Requarks/wiki/pull/2160
